app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt

```diff
@@ -429,16 +427,6 @@ class TalkTopicActivity : BaseActivity(), LinkPreviewDialog.Callback {
                 .show()
     }
 
-    override fun onLinkPreviewLoadPage(title: PageTitle, entry: HistoryEntry, inNewTab: Boolean) {
-        startActivity(if (inNewTab) PageActivity.newIntentForNewTab(this, entry, title) else
-            PageActivity.newIntentForCurrentTab(this, entry, title, false))
-    }
-
-    override fun onLinkPreviewAddToList(title: PageTitle) {
-        ExclusiveBottomSheetPresenter.show(supportFragmentManager,
-                AddToReadingListDialog.newInstance(title, Constants.InvokeSource.TALK_TOPIC_ACTIVITY))
```

Comment:
It looks like the `InvokeSource` should be `TALK_TOPIC_ACTIVITY` but now it will be `LINK_PREVIEW_MENU`. Should we pass the `InvokeSource` to the `LinkPreviewDialog` for analytics?

Reply:
I noticed this, but it was never consistent for all the other activities (everywhere else uses `LINK_PREVIEW_MENU`), so why not here, too? I am not aware of any analytics that depend on the invokeSource here.

Reply:
Ok, that makes sense to me.